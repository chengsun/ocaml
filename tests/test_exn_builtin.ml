let f x =
  raise Not_found

let g x =
  raise (Invalid_argument x)

let tryit fn x =
  try fn x; () with
  | Not_found -> Printf.printf "Not_found\n"
  | Invalid_argument y -> Printf.printf "Invalid_argument %s\n" y

let _ =
  tryit f "test";
  tryit g "test"
