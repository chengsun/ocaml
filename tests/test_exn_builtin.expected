Not_found
Invalid_argument test
